htsget-actix/README.md

```diff
@@ -50,7 +50,7 @@ are exposed in the public API.
 This crate has the following features:
 * `s3-storage`: used to enable `S3Storage` functionality.
 * `url-storage`: used to enable `UrlStorage` functionality.
-* `c4gh-experimental`: used to enable `C4GHStorage` functionality.
+* `experimental`: used to enable `C4GHStorage` functionality.
```

Comment:
I'd extend the semantics of this feature flag to "anything out of htsget official spec" also?

Reply:
Good point, I'll clarify that.